exercises/practice/isbn-verifier/.meta/tests.toml

```diff
@@ -48,7 +55,10 @@ description = "empty isbn"
 description = "input is 9 characters"
 
 [ed6e8d1b-382c-4081-8326-8b772c581fec]
-description = "invalid characters are not ignored"
+description = "invalid characters are not ignored after checking length"
```

Comment:
Is it important that it is not ignored before or after checking the length?   Or is it only important that the valid ISBN's are coming back as vaildated?  This seems like it is a test that will suggest a specific order of solution, rather than only verify a solution.

Reply:
Indeed, some solutions may not only not ignore the invalid characters, but may seek them out and strip them.  I am not sure this is a good test as described now.

Reply:
This change and the other (and all the others from the "Sync" batch of PRs) was done by `configlet sync`. Everything comes from `problem-specifications`, if you want to make another change to needs to be from there.

Reply:
I thought at first, until I saw the title change, then I figured it was a suggested change.

It is one that I do not think I would have my track opt into.  But it may be important for Elixir.

Reply:
I  can not cancel my review for some reason (the request to change... looking into it now.)

Reply:
This comes from [problem-specifications](https://github.com/exercism/problem-specifications) (link provided) if someone else would like to go there to check it out with a click.